src/myspace/AdditionalInfo.jsx

```diff
@@ -87,31 +87,20 @@ export default function AdditionalInfo() {
           let data = await response.json();
           if (response.ok)
             return setResponseMessage(data.message)
-          if (response.status === 409)
-            return setIsExist(true);
-          setErrorMessage(data.message);
+          return setResponseMessage(data.message);
         })
-        .catch(() => setErrorMessage("The service is temporarily unavailable, please try again later."));
+        .catch(() => setResponseMessage(SERVICE_UNAVAILABLE_ERROR));
     }
   }
 
```

Comment:
shouldn't it be update all the time? 
why does it try to create and then if conflict update? If it's on this page, it should always try to update correct?

Reply:
@meenakshi-dhanani . Because when user retrieve the page for the first time, the additional info would have not yet been created. The first response that user will see is `No additional information found with your data. Please provide them now." Then users will need to fill their additional info. The first time they click `Save` button, this will send POST /user/additional_info request to backend. 
But since wee don't know which login user will already have created an additional information or not, every time user go to this page and fill in the information,  on the `Save` button click the system would by default try to send POST request, which will return error message 409 Conflict, then will automatically send PUT request to continue with update information. Does this make sense?